docs/services/vision/detection.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Detection (or 2D object detection)"
+linkTitle: "Detection"
+weight: 10
+type: "docs"
+description: "Select an algorithm that identifies objects in a 2D image and adds bounding boxes around identified objects."
+tags: ["vision", "computer vision", "CV", "services", "detection"]
+# SMEs: Bijan, Khari
+---
+
+_2D Object Detection_ is the process of taking a 2D image from a camera and identifying and drawing a box around the distinct "objects" of interest in the scene.
+Any camera that can return 2D images can use 2D object detection.
+
+The service provides different types of detectors, both heuristic and machine-learning based, so that you can create, register, and use detectors for any object you may need to identify.
```

Comment:
```suggestion
The service provides different types of detectors, both heuristic and machine-learning-based, so that you can create, register, and use detectors for any object you may need to identify.
```

Reply:
```suggestion
The service provides different types of detectors, both based on heuristics and machine learning, so that you can create, register, and use detectors for any object you may need to identify.
```